LayoutTests/inspector/css/generateFormattedText-expected.txt

```diff
@@ -50,8 +50,8 @@ a: 1; b: 2; c: 3;
 a: 1; b: 2; c: 3;
 
 -- Running test case: CSS.generateFormattedText.MatchedRules.includeGroupingsAndSelectors
-@media only screen and (min-width: 0px) {@media only screen and (min-height: 0px) {body > div#test-node {a: 1; b: 2; c: 3;}}}
```

Comment:
odd that this changed 🤔

i wouldn't expect `WI.CSSFormatter` to be affected by any changes to `WI.CSSGrouping` (since IIRC it runs in a `Worker`).  any idea why this happened, and maybe a way to prevent that?

Reply:
ooohhh oops this is `WI.CSSStyleDeclaration.prototype.generateFormattedText`, not `WI.CSSFormatter`

yeah this is totally fine to change then :)